man/systemd.xml

```diff
@@ -204,6 +204,13 @@
         <option>info</option>,
         <option>debug</option>.</para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term><option>--log-time=</option></term>
+
+        <listitem><para>Prefix log messages with timestamp. Argument is
+        a boolean value. If the argument is omitted, it defaults to
```

Comment:
Please make it default to true, like `--log-color` below. The idea is that `systemd --log-time` means "yes", not "no".

Reply:
ack